internal/cmd/generator/errors.json

```
@@ -5,6 +5,12 @@
         "http": 400,
         "permanent": true
     },
+    "RequestEntityTooLargeError": {
+        "description": "It means that the request has too large enities.",
+        "grpc": 14,
```

```suggestion
        "grpc": 11,
```

https://grpc.github.io/grpc/core/md_doc_statuscodes.html OUT_OF_RANGE = 11